### PR TITLE
fix: more broken links

### PR DESCRIPTION
**Pull request by fzipi**

### CODE REVIEW

File: content/blog/2020-09-01-introducing-msc_pyparser.md [hunk 1]
```diff
@@ -33,20 +33,20 @@ pip3 install msc-pyparser==0.3
 
 This gives you the `msc_parser` library for use in python script in the version 0.3.
 
-To get started it's useful to also grab crs\_read.py and crs\_write.py from the examples folder of the `msc_parser` github:
+To get started it's useful to also grab `crs_read.py` and `crs_write.py` from the examples folder of the `msc_parser` github:
```
Comment (theseion) [inline content/blog/2020-09-01-introducing-msc_pyparser.md:36]:
```suggestion
To get started it's useful to also grab `crs_read.py` and `crs_write.py` from the examples folder of the `msc_pyparser` github:
```

File: content/blog/2020-09-01-introducing-msc_pyparser.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ And this gives you the rule set in the new folder `coreruleset/rules-new`.
 If you want, you can check the original rules and the new copy is identical:
 
 ```sh
```
Comment (theseion) [inline content/blog/2020-09-01-introducing-msc_pyparser.md:86]: `bash-session`?

File: content/blog/2020-09-01-introducing-msc_pyparser.md [hunk 3]
```diff
@@ -170,9 +170,9 @@ With the script executing successfully, you can now export the rules in rules-ya
 
 ### Example 2 - Add new tags to certain rules
 
-Maybe you have read the blog post about the plans to [introduce CAPEC tagging into CRS](https://coreruleset.org/20200608/overhauling-the-crs-tags/). We have implemented these changes (and it's already in the CRS v3.3 release) with the help of the `msc_parser`. We prepared the CAPEC tag for every rule in a seperate CSV file (-&gt; format: rule-id;tag).
+Maybe you have read the blog post about the plans to [introduce CAPEC tagging into CRS]({{< ref "blog/2020-06-08-overhauling-the-crs-tags.md" >}}). We have implemented these changes (and it's already in the CRS v3.3 release) with the help of the `msc_parser`. We prepared the CAPEC tag for every rule in a seperate CSV file (-> format: rule-id;tag).
```
Comment (theseion) [inline content/blog/2020-09-01-introducing-msc_pyparser.md:173]:
```suggestion
Maybe you have read the blog post about the plans to [introduce CAPEC tagging into CRS]({{< ref "blog/2020-06-08-overhauling-the-crs-tags.md" >}}). We have implemented these changes (and it's already in the CRS v3.3 release) with the help of the `msc_pyparser`. We prepared the CAPEC tag for every rule in a seperate CSV file (-> format: rule-id;tag).
```

File: content/blog/2020-09-01-introducing-msc_pyparser.md [hunk 4]
```diff
@@ -397,7 +397,9 @@ if __name__ == "__main__":
 
 And here is how to call the script:
 
-`$> remove_rule_by_id.py /tmp/coreruleset/rules-yaml/REQUEST-920-PROTOCOL-ENFORCEMENT.yaml /tmp/coreruleset/rules-yaml-new/REQUEST-920-PROTOCOL-ENFORCEMENT.yaml 920180`
+```sh
```
Comment (theseion) [inline content/blog/2020-09-01-introducing-msc_pyparser.md:400]: `bash-session`?

**Review by theseion — CHANGES_REQUESTED** [review]

File: content/blog/2020-09-01-introducing-msc_pyparser.md (hunk 2 above)
Comment (fzipi) [reply content/blog/2020-09-01-introducing-msc_pyparser.md:86]: If it is one line, there is no difference in the highlight mode.

File: content/blog/2020-09-01-introducing-msc_pyparser.md (hunk 4 above)
Comment (fzipi) [reply content/blog/2020-09-01-introducing-msc_pyparser.md:400]: If it is one line, there is no difference in the highlight mode.

**Review by theseion — APPROVED** [review]

**Pull request merged**